lib/generators/rolemodel/css/base/templates/app/javascript/stylesheets/variables.scss

```diff
@@ -47,6 +47,9 @@
   --font-primary: 'Roboto', sans-serif;
 
   // Text Ratios
+  // This is using a 16px base font-size. The scale ratio can be changed to
```

Comment:
What ratio are we using?

Reply:
@mark-kraemer It is the minor third scale if you want to document that.